mixins/loading-complete/loading-complete-mixin.js

```diff
@@ -0,0 +1,21 @@
+export const LoadingCompleteMixin = (superclass) => class extends superclass {
+
+	constructor() {
+		super();
+		this._loadingCompletePromise = new Promise(resolve => {
+			this._loadingCompleteResolve = resolve;
+		});
+	}
+
+	getLoadingComplete() {
+		return this._loadingCompletePromise;
+	}
+
+	loadingComplete() {
```

Comment:
in the interest of bikeshedding over semantics is there a more descriptive name we could be using for this? Some other ideas I was workshopping:
- `completeLoad`
- `resolveLoadingComplete`
- `setLoadingComplete`
- `markLoadingComplete`

Reply:
Yeah my only hesitation with `loadingComplete` is in Lit, it has `updateComplete` which is the `Promise` that resolves and `getUpdateComplete()` which is the method you can override to customize it. We kind of borrowed those conventions with `getLoadingComplete()`, so having `loadingComplete` be a method deviates a bit. I kind of like `resolveLoadingComplete()` but curious what others think...

Reply:
Same. I like `resolveLoadingComplete` feels like it avoids the possible confusion with the `updateComplete`